pkg/sdk/object_types.go

```diff
@@ -25,6 +25,7 @@ const (
 	ObjectTypeNetworkPolicy        ObjectType = "NETWORK POLICY"
 	ObjectTypePasswordPolicy       ObjectType = "PASSWORD POLICY"
 	ObjectTypeSessionPolicy        ObjectType = "SESSION POLICY"
+	ObjectTypePrivacyPolicy        ObjectType = "PRIVACY POLICY"
```

Comment:
sdk.ObjectType is used in more places than just tagging, are we sure, that we do not break other resources by adding it (e.g. grants)?

Reply:
We check tag object types in https://github.com/Snowflake-Labs/terraform-provider-snowflake/blob/tag-v1/pkg/sdk/grants_validations.go#L21 and https://github.com/Snowflake-Labs/terraform-provider-snowflake/blob/tag-v1/pkg/resources/grant_ownership.go#L409.  Fortunately, each current test passed.